typescript/cli/src/commands/warp.ts

```diff
@@ -438,7 +450,7 @@ export const rebalancer: CommandModuleWithContext<{
     );
 
     // Instantiates the strategy that will get rebalancing routes based on monitor results
-    const strategy: IStrategy = new Strategy();
+    const strategy: IStrategy = new Strategy(BigInt(strategyTolerance));
```

Comment:
shall we let the user know that they should express the tolerance in integer and in "wei" (or whatever the amount of decimals that the tokens have)?

Or this is too obvious to be explained?

Reply:
Added it to the command description https://github.com/BootNodeDev/hyperlane-monorepo/pull/29/commits/45d956d1974c2ef6b44d50a265cee819f608066b